pkg/security/tests/process_test.go

```diff
@@ -361,6 +381,13 @@ func TestProcessContext(t *testing.T) {
 	})
 
 	test.Run(t, "args-overflow-single", func(t *testing.T, kind wrapperType, cmdFunc func(cmd string, args []string, envs []string) *exec.Cmd) {
+		if kind == dockerWrapperType && test.opts.staticOpts.enableEBPFLess == true {
```

Comment:
Per the "if" below it this that we skip whatever the "kind". I would suggest to remove this one

Reply:
The first `if` is to skip ebpfless docker tests, the second is temporary until we add what's missing ebpfless side to make it pass (and still, only on host).